app/components/candidate_interface/course_choices_review_component.rb

```diff
@@ -186,6 +186,8 @@ def status_row(application_choice)
     end
 
     def offer_withdrawal_reason_row(application_choice)
+      return nil unless application_choice.offer_withdrawn?
```

Comment:
Maybe simpler if this guard clause was combined with the `if` below it? e.g.
```
      return nil unless application_choice.offer_withdrawn? && application_choice.offer_withdrawal_reason.present?
```
Not sure.